chapters/apps.tex

```diff
@@ -154,7 +154,7 @@ \section{Separating Agreement and Execution}
 This sort of optimistic execution is the focus of an approach to scalable BFT systems 
 that can work quite well for particular applications where conflicts between transactions are rare. 
 At the same time, it adds additional complexity to the client, by virtue of needing to handle possibly invalid results.
-The approach is discussed further in Chapter \ref{ch:related}.
+The approach is discussed further in Chapter \ref{ch:related}.\looseness=-1
```

Comment:
Currently the 10 from Chapter 10 is on its own page. This hackery just moves it up.